### ISSUE TITLE
views_simplecache is not created

### ISSUE THREAD

**Issue by diybl** [Elgg/Elgg#13427]
3.3.16

i click the flush cache, also not created.

my data/caches
only create the two dir:
localstash
stash

not has 
views_simplecache

**Comment by jdalsem** [comment]
Could this be a duplicate of https://github.com/Elgg/Elgg/issues/12565

**Comment by diybl** [comment]
> Could this be a duplicate of #12565

no i find the url is:
http://xxxxxxxx/cache/1617335643/default/font-awesome/css/all.min.css
not exist..

the data folder also has not the folder

**Comment by jdalsem** [comment]
is your cache location symlinked?

**Comment by daniyaree** [comment]
If there is something like Cache Manager plugin or anything similar to it in your web hosting panel see  "Advanced Options" or similar options and disable "Accelerate static content". Purging all cache or setting all cache plugins / cache systems / whatever cache is there  may be needed in addition. Once this is done "views_simplecache" will be generated. 
Its important to note that even with Cache / Cache Managers disabled it is still advised : "If the website .... uses application with dynamic routing, we recommend disabling this option." A lot of hosting panels or systems are using various CACHE systems or plugins or modules to speed up website - but this adds to problem like described above, and also confuses a hell out of me : what happens when there are constant comments (as in a live social site) and constant dynamic updates like "Likes" "Notifications" or if commercial site "Purchases"  .. its tricky HOW to set up enable Caching or not enable it (which can make the site slow) or strike the golden middle path. The options coming with hosting web server OS, php and stack considerations as well as the php script considerations (Elgg in this case) are needed. Also see this : http://learn.elgg.org/en/2.0/intro/install/nginx.html

**Comment by jdalsem** [comment]
@diybl did you fix your issue?
